src/components/Book.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({ title, author }) {
+  return (
+    <span>
+      {title}
+      by
+      {author}
+    </span>
+  );
+}
```

Comment:
- [ ] This component is expected to have a button to remove a book. It's expected that a book has a button that when clicked, deletes a book from the list and the button is best to be placed in this component. Kindly add that 👍🏽

Reply:
I add a button as suggested